load-dataset.py

```diff
@@ -1,3 +1,10 @@
 from datasets import load_dataset
 
 import os
+
+
+dir_name = "../data/150-percent"
+
+print("Checking that the path exists...:",os.path.exists(dir_name))
```

Comment:
I would just add a space after the colon for output :)
```suggestion
print("Checking that the path exists...: ",os.path.exists(dir_name))
```